### PR TITLE
Feature/non mac super symbol

### PR DESCRIPTION
**Pull request by toaster**
### Description:

This PR introduces a different rune representing the Super key on non-Apple systems, the “BLACK DIAMOND MINUS WHITE X” (`0x2756`, ❖) as fake Windows logo.

It also includes some missing documentation comments and replacement of deprecated constants.

### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

#### Where applicable:

- [x] Any breaking changes have a deprecation path or have been discussed.

### CODE REVIEW

File: key.go [hunk 1]
```diff
@@ -176,15 +176,40 @@ const (
...
+	//
+	// Since: 2.2
 	KeyModifierSuper
 )
+
+const (
+	// RuneModifierAlt is the rune that represents the Alt key.
+	//
+	// Since: 2.2
+	RuneModifierAlt = '⌥'
```
Comment (andydotxyz) [inline key.go:206]: Do these need to be public? Isn’t it a rendering detail rather than part of the user API?

**Review by andydotxyz — COMMENTED** [review]
Looks great, but not sure I understand adding APIs for the runes we use in the menu…

File: key.go (hunk 1 above)
Comment (toaster) [reply key.go:206]: Yes, you’re right.

**Review by andydotxyz — APPROVED** [review]
Nice thanks

**Pull request merged**